README.md

```diff
@@ -882,7 +882,57 @@ constructor(props) {
 
 ### Angular
 
-> TODO
+Since Angular 4.0.0, alongside standard ngIf, it is possible to use ngIf;else or ngIf;then;else using `<ng-template>` with an alias `#aliasName`.
+
+ngIf;else example:
```

Comment:
I'd use only `ngIf` in the first example, move the sentence above below the snippet and there show `ngIf;then;else` as it is now. What do you think? Also use `` ` `` for the directive names.